lang/nl.js

```diff
@@ -108,7 +108,7 @@ export default {
 	"components.tag-list.show-more-description": "Selecteer deze optie om verborgen items op labellijsten weer te geven",
 	"components.tag-list-item.tooltip-arrow-keys": "Pijltoetsen",
 	"components.tag-list-item.tooltip-arrow-keys-desc": "Schakelen tussen tags",
-	"components.tag-list-item.tooltip-delete-key": "Backspace/Delete",
+	"components.tag-list-item.tooltip-delete-key": "Backspace/Verwijderen",
```

Comment:
So close and yet so far away.

Reply:
According to Google, "Backspace" in Dutch is actually "Backspace"... I'd ask my Opa but I'm fairly certain he's never had to use that word 😛

Reply:
lol whoa, that's neat! I stand corrected.